azure-communication-ui/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     ext {
-        call_library_version_name = '1.11.0'
+        call_library_version_name = '1.13.0-beta.1'
```

Comment:
We should really be using the beta entry for SDK, and using flags to gate Beta API if necessary.

Reply:
This is UI lib version, not SDK